### ISSUE TITLE
Port mangling function to DerelictUtil?

### ISSUE THREAD

**Issue by p0nce** [DerelictOrg/DerelictUtil#5]
As you know @aldacron, DerelictFI has that unholy function that builds decorated names for the Windows 32-bits case: https://github.com/DerelictOrg/DerelictFI/blob/master/source/derelict/freeimage/freeimage.d#L57

Now FANN has the same decorated names so I had to copy/paste this trick for `derelict_extras-fann`: https://github.com/p0nce/DerelictFANN/commit/2f52b644df02f4d3ab8fd2b91275f8dc7695a45d

Maybe it could be in DerelictUtil once we have a third case?

**Comment by mdparker** [comment]
So two years later(!!!), you might be interested to know that I've finally implemented this in SharedLibLoader. I'm working on a DerelictUtil 3.0, with a bit of a revamp planned across every package. bindFunc is now public, and so is the new [bindFunc_stdcall](https://github.com/DerelictOrg/DerelictUtil/blob/2.1/source/derelict/util/loader.d#L90) in the misnamed 2.1 branch.

**Comment by p0nce** [comment]
Kewl. Is v3 backwards compatible, so that `>=` was sensible? I've used `~>` out of learned habit.

**Comment by mdparker** [comment]
It will be source compatible as long as every package that depends on DerelictUtil in the same project is importing from the 3.0 release. Binary compatibility with previous versions is not possible, I think. I was initially going to tag it 2.1, but I want to be very clear that this is different from previous versions.

`~>` is definitely the way to go. I learned that lesson well when I upgraded to DerelictUtil 2.0. This time, I'm not tagging a release until I have a branch in every Derelict package using it and ready to be tagged with it.
